packages/manager/apps/public-cloud/src/components/walkMe/translations/Messages_fr_FR.json

```
@@ -0,0 +1,6 @@
+{
+  "walkme_title": "Nouveau",
+  "walkme_content_server": "<strong>Votre menu a évolué pour vous faire gagner du temps !</strong> Retrouvez notamment vos produits VPS, serveurs dédiés et Private Cloud",
```

```suggestion
  "walkme_content_server_1": "Votre menu a évolué pour vous faire gagner du temps !",
  "walkme_content_server_2": "Retrouvez notamment vos produits VPS, serveurs dédiés et Private Cloud",
```
?

this way you could remove the HTML marking :)

---

Would be much more complicated to handle as the content varies depending on the attached element, the screen width etc